benchexec/tools/avr.py

```diff
@@ -15,13 +15,21 @@ class Tool(benchexec.tools.template.BaseTool2):
     URL: https://github.com/aman-goel/avr
     """
 
+    REQUIRED_PATHS = ["build/"]
```

Comment:
A directory with this name looks as if it contains build-time stuff, which is typically not required at runtime. Are you sure that this is not the case here?

Reply:
Yes, the build script of AVR by default places the compiled executables under `build/` and the intermediate object code (`*.o`) under `src/`.